doc/sphinx/tutorial/write-your-first-wayland-compositor.md

```diff
@@ -61,13 +64,17 @@ int main(int argc, char const* argv[])
 }
 ```
 
-This program creates a floating window manager with basic window management
-capabilities such as controlling multiple windows, minimizing and maximizing,
-and handling mouse input. This is done with the help of MirAL (Mir Abstraction
-Layer) which gives you a high level interface to work with Mir.
 
-### Building
-To compile this simple program, you can use this `CMakeLists.txt` file:
+`MirRunner` is a class from the `libmiral` library that that provides a  high level interface for  Mir.
+
+`MirRunner runner` calls `run_with()` which specifies different fucntions of the composer. In this example, `run_with()` evokes a function `set_window_management_policy` that applies `MinimalWindowManager` policy to `runner`. The composer is therefore created with basic window management capabilities such as controlling multiple windows, minimizing and maximizing, and handling mouse input. 
```

Comment:
```suggestion
`MirRunner runner` creates a `runner` object that can be used to configure your compositor.  To run the compositor you call `runner.run_with(...)` which specifying different functions of the composer. In this example, `run_with()` is passed a function `set_window_management_policy` that applies `MinimalWindowManager` policy to `runner`. The composer is therefore created with basic window management capabilities such as controlling multiple windows, minimizing and maximizing, and handling mouse input. 
```

Reply:
+1 on both comments, my version would be something like:

```suggestion
`MirRunner runner` creates a `runner` object that can be used to configure your compositor.  To run the compositor you call `runner.run_with(...)`, passing in different components to customize the behavior of the compositor. In this example, `run_with()` is passed a function `set_window_management_policy` that applies a `MinimalWindowManager` policy to the compositor. The compositor is therefore created with basic window management capabilities such as controlling multiple windows, minimizing and maximizing, and handling mouse input. 
```